ugvc/mrd/mrd_utils.py

```diff
@@ -169,10 +169,18 @@ def collect_coverage_per_locus(coverage_bw_files, df_sig):
     return df_list
 
 
+def collect_coverage_per_locus_gatk(coverage_csv, df_sig):
+    coverage_gatk = pd.read_csv(coverage_csv, usecols=["Chrom", "Pos", "Total_Depth"])
+    coverage_gatk.rename(columns={"Chrom": "chrom", "Pos": "pos", "Total_Depth": "coverage"}, inplace=True)
+    coverage_gatk.set_index(["chrom", "pos"], inplace=True)
+    df_sig = df_sig.join(coverage_gatk, how="left")
```

Comment:
Please make sure the index of df_sig is indeed ["chrom", "pos"], if not raise an exception